### ISSUE TITLE
[Accessibility] PeoplePicker doesn't have focus border when focused by keyboard

### ISSUE THREAD

**Issue by ipip2005** [OfficeDev/office-ui-fabric-react#999]
### Bug Report

- __Package version(s)__: latest
- __Browser and OS versions__: any

#### Describe the issue:

The PeoplePicker control doesn't have focus border when TAB focused by keyboard.

#### Actual behavior:

No focus border

#### Expected behavior:

A focus border around the box.

### If applicable, please provide a codepen repro:

https://fabricreact.azurewebsites.net/fabric-react/master/#/examples/PeoplePicker

**Comment by micahgodbolt** [comment]
Current experience is that "focused" item has dark background. Is focus border expected instead?

![image](https://cloud.githubusercontent.com/assets/1434956/23033652/d179d5ca-f42d-11e6-99ba-f65f63cedcba.png)

**Comment by ipip2005** [comment]
The focus border I meant is actually the border of the input box.
Like I saw in TextField:

![image](https://cloud.githubusercontent.com/assets/7711735/23297148/523c79de-fab3-11e6-9102-4584cd4d8d26.png)
